plugins/prefetch/pattern.h

```diff
@@ -23,22 +23,16 @@
 
 #pragma once
 
-#ifdef HAVE_PCRE_PCRE_H
-#include <pcre/pcre.h>
-#else
-#include <pcre.h>
-#endif
-
 #include "common.h"
+#include "tsutil/Regex.h"
 
 /**
  * @brief PCRE matching, capturing and replacing
  */
 class Pattern
 {
 public:
-  static const int TOKENCOUNT = 10;             /**< @brief Capturing groups $0..$9 */
-  static const int OVECOUNT   = TOKENCOUNT * 3; /**< @brief pcre_exec() array count, handle 10 capture groups */
+  static const int TOKENCOUNT = 10; /**< @brief Capturing groups $0..$9 */
 
   Pattern();
   virtual ~Pattern();
```

Comment:
Is inheritance actually used here? If not, let's remove the `virtual`.

Reply:
Removed the custom destructor.